File: AlphaStream/Requests/GetAlphaTagsRequest.py
class GetAlphaTagsRequest(object):
    """ Fetch all alpha tags and the number of alphas tagged with each. Tags can be used in SearchAlphas() """

    def __init__(self):
        self.Endpoint = "alpha/tags/read"

    def GetPayload(self):
        """ Construct search query data payload from the initialized properties """
        payload = {}

        return payload

File: QuantConnect.AlphaStream/Requests/GetAlphaTagsRequest.cs
using QuantConnect.AlphaStream.Infrastructure;
using QuantConnect.AlphaStream.Models;
using RestSharp;
using System.Collections.Generic;

namespace QuantConnect.AlphaStream.Requests
{
    /// <summary>
    /// Fetch number of alphas containing each tag
    /// </summary>
    [Endpoint(Method.GET, "alpha/tags/read")]
    public class GetAlphaTagsRequest : AttributeRequest<List<Tag>>
    {
    }
}
